package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jupiterone/graph-automox",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
Can you remove this and delete the remote tag? We use [Auto](https://github.com/intuit/auto) for publishing. You can add a `patch` and `release` label to this PR before merging. Once merged, it should publish the package.